python/open3d/visualization/gui/__init__.py

```diff
@@ -25,8 +25,8 @@
 # ----------------------------------------------------------------------------
 """Functionality for quickly building Graphical User Interfaces."""
 
-if "@BUILD_GUI@" == "ON":
-    import open3d
+import open3d
```

Comment:
Same, if `@@` configuration is no longer needed, we can remove this file from `cpp/pybind/make_python_package.cmake`.

Reply:
Actually removed `__init__.py` for pure pybind code (visualization, rendering and gui), since direct insertion of names works. If BUILD_GUI is False, the user will get a straight `ModuleNotFoundError: No module named 'open3d.visualization.gui`

Reply:
CI was failing due to BUNDLED OPen3D-ML Python code. This is fixed by inserting pybind symbols in two stages.